locales/en/app.ftl

```diff
@@ -446,3 +446,8 @@ fxa-what-to-do-blurb-4 =
   on your phone and computer so you don’t have to remember them all.
 
 fb-landing-headline = Was your info exposed in the { $breachName } data breach?
+
+copyright = 
+  Portions of this content are { $years } by individual mozilla.org contributors.
```

Comment:
Can we have only the final year exposed as a variable? Right now the code is making a lot of assumptions on spacing, separators, etc.

```
copyright = 
  Portions of this content are © 1998-{ $year } by individual mozilla.org contributors.
```

Reply:
Yep, will do!